File: web_scraping.rb
# http://rosettacode.org/wiki/Web_scraping

require 'net/http'

def get_site(url)
  Net::HTTP.get URI(url)
end

get_site("http://tycho.usno.navy.mil/cgi-bin/timer.pl").each_line do |line|
  puts line if line.include?("UTC") 
end    

File: web_scraping.js
#!/usr/bin/env node
// http://rosettacode.org/wiki/Web_scraping

var http = require('http');

var options = {
  host: 'tycho.usno.navy.mil',
  path: '/cgi-bin/timer.pl'
}

callback = function(response) {
  var str = '';

  response.on('data', function(chunk) {
    str += chunk;
  });

  response.on('end', function () {
    var arr = str.split("<BR>");
    console.log(arr[1])
  });
}

http.request(options, callback).end();
